### PR TITLE
API-39: Add integration tests on select attributes validation

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/AbstractAttributeTestCase.php [hunk 1]
```diff
@@ -140,6 +140,27 @@ public function assertDoesNotHaveAReferenceDataName($type)
         $this->assertSame('reference_data_name', $violations->get(0)->getPropertyPath());
     }
 
+    public function assertDoesNotHaveMaxCharacters($type)
```
Comment (skeleton) [inline src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/AbstractAttributeTestCase.php]: testAssert

File: src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/MultiSelectIntegration.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    public function testMultiSelectShouldNotHaveADateMin()
+    {
+        $this->assertDoesNotHaveADateMin('pim_catalog_multiselect');
+    }
+
+    public function testMultiSelectShouldNotHaveADateMax()
+    {
+        $this->assertDoesNotHaveADateMin('pim_catalog_multiselect');
```
Comment (skeleton) [inline src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/MultiSelectIntegration.php]: assertDoesNotHaveADateMax ?

File: src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/SimpleSelectIntegration.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    public function testSimpleSelectShouldNotHaveADateMin()
+    {
+        $this->assertDoesNotHaveADateMin('pim_catalog_simpleselect');
+    }
+
+    public function testSimpleSelectShouldNotHaveADateMax()
+    {
+        $this->assertDoesNotHaveADateMin('pim_catalog_simpleselect');
```
Comment (skeleton) [inline src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/SimpleSelectIntegration.php]: same

File: src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/AbstractAttributeTestCase.php (hunk 1 above)
Comment (phaseinducer) [inline src/Pim/Bundle/CatalogBundle/tests/integration/Validation/Attribute/AbstractAttributeTestCase.php]: assert is ok, it's just the refacto in the abstract. btw it should be protected.

**Pull request merged**